src/Controller/CRUDController.php

```diff
@@ -1563,7 +1563,7 @@ private function setFormTheme(FormView $formView, ?array $theme = null): void
 
     private function handleXmlHttpRequestErrorResponse(Request $request, FormInterface $form): ?JsonResponse
     {
-        if (!\in_array('application/json', $request->getAcceptableContentTypes(), true)) {
+        if (0 === \count(array_intersect(['application/json', '*/*'], $request->getAcceptableContentTypes()))) {
             @trigger_error('In next major version response will return 406 NOT ACCEPTABLE without `Accept: application/json`', E_USER_DEPRECATED);
```

Comment:
IMO, the deprecation message should reflect this change.

Reply:
I agree. The change is now reflected in the deprecation message.